admin/templates/registration_providers/edit_moderators.html

```diff
@@ -22,30 +22,40 @@ <h2>{{ registration_provider.name }}</h2>
         </div>
         <div class="row">
             <div class="col-md-12">
-                <form id="add-moderators-form" action="{% url 'registration_providers:add_moderators' registration_provider.id %}" method="post">
+                <form id="add-moderators-form" action="{% url 'registration_providers:add_admin_or_moderator' registration_provider.id %}" method="post">
                     {% csrf_token %}
                     <label>Add moderator by guid: </label>
                     <input type="text" name="add-moderators-form">
-                    <input type="submit" value="Add Moderator" class="form-button btn btn-success">
+                    <input type="submit" name="mod" value="Add Moderator" class="form-button btn btn-success">
+                    <input type="submit" name="admin" value="Add Admin" class="form-button btn btn-success">
                 </form>
             </div>
         </div>
         <hr>
         <div class="row">
             <div class="col-md-12">
-                <form id="remove-moderators-form" action="{% url 'registration_providers:remove_moderators' registration_provider.id %}" method="post">
+                <form id="remove-moderators-form" action="{% url 'registration_providers:remove_admins_and_moderators' registration_provider.id %}" method="post">
                     {% csrf_token %}
                     <table class="table table-striped">
                         <th></th>
                         <th>Name</th>
+                        <th>Type</th>
                         {% for moderator in moderators %}
                             <tr>
-                                <td><input type='checkbox' name="{{moderator.id}}"></td>
+                                <td><input type='checkbox' name="Moderator-{{moderator.id}}"></td>
```

Comment:
Can you just make this a non-name HTTP tag attribute and have the that be the thing that distinguishes them? Like `<input type='checkbox' name="{{moderator.id}}" group="moderator"></td>` ?

Reply:
It looks like the group attribute isn't passed off in the request.